docs/docs/index.md

```diff
@@ -5,52 +5,54 @@ Welcome to zamba's documentation!
 <div class="embed-responsive embed-responsive-16by9" width=500>
     <iframe width=600 height=340 class="embed-responsive-item" src="https://s3.amazonaws.com/drivendata-public-assets/monkey-vid.mp4" frameborder="0" allowfullscreen=""></iframe>
 </div>
+<!-- TODO: what is the above supposed to do? it doesn't look like it's working><!-->
 
 *Zamba means "forest" in the Lingala language.*
 
 Zamba is a command-line tool built in Python to automatically identify the
-species seen in camera trap videos from sites in central Africa. Using the
+species seen in camera trap videos from sites in central and west Africa. Using the
 combined input of various deep learning models, the tool makes predictions
-for 23 common species in these videos (as well as blank, or, "no species
+for 31 common species in these videos (as well as blank, or, "no species
 present").
 
+*New in Zamba v2:* Zamba now has an additional model trained on 11 common European species. <!--TODO: add more details about where from><!-->
+
 # Quickstart
 
 This section assumes you have successfully installed `zamba` and want to get
-right to making species predictions for some videos! All of the commands here should be run at the commandline. On
-macOS, this can be done in the terminal (⌘+space, "Terminal"). On Windows, this can be done in a command prompt, and if you installed Anaconda an anaconda prompt (Start > Anaconda3 > Anaconda Prompt).
+right to making species predictions for some videos! All of the commands here should be run at the command line. On
+macOS, this can be done in the terminal (⌘+space, "Terminal"). On Windows, this can be done in a command prompt, or if you installed Anaconda an anaconda prompt (Start > Anaconda3 > Anaconda Prompt).
 
 ## Input videos
 
-For example, we might have the directory `vids_to_classify` with camera trap videos in it:
+For example, we might have the directory `vids_to_classify` with camera trap videos in it. Let's list the videos:
 
 ```console
 $ ls vids_to_classify/
-blank1.mp4
-blank2.mp4
+blank.mp4
+chimp.mp4
 eleph.mp4
-small-cat.mp4
-ungulate.mp4
+leopard.mp4
 ```
-
+<!-- TODO: possibly update with newer videos><!-->
 Here are some screenshots from those videos:
```

Comment:
What is the best way to resize videos without losing too much quality? When I  used Tools > Adjust size in preview to get each png down to ~30 KB, the quality is really really bad. For now the full size images are used as placeholders

Reply:
videos or images? https://imageoptim.com/mac is generally pretty easy and works